kasa/exceptions.py

```diff
@@ -27,9 +28,15 @@ class RetryableException(SmartDeviceException):
     """Retryable exception for device errors."""
 
 
-class TimeoutException(SmartDeviceException):
+class TimeoutException(SmartDeviceException, TimeoutError):
     """Timeout exception for device errors."""
 
+    def __repr__(self):
+        return SmartDeviceException.__repr__(self)
+
+    def __str__(self):
+        return SmartDeviceException.__str__(self)
```

Comment:
I don't think we should override these.

Reply:
If we don’t it uses the `asyncio.TimeoutError` implementation which hides the inner exception. This way it forces consistency with the rest of the custom exceptions.

Reply:
Ok, let's just merge it then :-)